### PR TITLE
ACCEL-554: Adds relationship between course cohort and course access …

### PR DESCRIPTION
**Pull request by NikolayBorovenskiy**
**Description:** Adds a relationship between course cohort and course access role.

**Youtrack:** https://youtrack.raccoongang.com/issue/ACCEL-554

**Reviewers:**
- [@idegtiarov ]

**Post merge:**
- [+] Delete working branch (if not needed anymore)

### CODE REVIEW

File: lms/djangoapps/instructor/models.py [hunk 1]
```diff
@@ -1,19 +1,45 @@
...
     class Meta:
         app_label = 'instructor'
-        unique_together = (('cohort', 'user'), )
+        unique_together = (('cohort', 'user'),)
+
+
+@receiver(post_save, sender=CohortAssigment)
+def allow_course_access_for_cohort_leader(sender, instance, created, **kwargs):  # pylint: disable=unused-argument
+    """
+    Send request to allow user access to course modification after the user is assigned as cohort leader of the course.
```
Comment (idegtiarov) [inline lms/djangoapps/instructor/models.py]: Too long docstring it should be no longer than 79 chars.

Comment (NikolayBorovenskiy) [reply lms/djangoapps/instructor/models.py]: Yep. Edx style guidelines. My fault and pushed a commit to improve.

File: lms/djangoapps/instructor/models.py [hunk 2]
```diff
@@ -1,19 +1,46 @@
...
     class Meta:
         app_label = 'instructor'
-        unique_together = (('cohort', 'user'), )
+        unique_together = (('cohort', 'user'),)
+
+
+@receiver(post_save, sender=CohortAssigment)
+def allow_course_access_for_cohort_leader(sender, instance, created, **kwargs):  # pylint: disable=unused-argument
+    """
+    Send request to allow user access to course modification
```
Comment (idegtiarov) [inline lms/djangoapps/instructor/models.py]: I'm sorry to say that, but this change is also not perfect, Multiline docstrings should have a title which is separated from the main text by an empty line )))

Comment (NikolayBorovenskiy) [reply lms/djangoapps/instructor/models.py]: oh gosh.. your truth.
Done))

**Pull request merged**